include/dlaf/tune.h

```diff
@@ -17,10 +17,13 @@ namespace dlaf {
 /// DLA-Future tuning parameters.
 ///
 /// Holds the value of the parameters that can be used to tune DLA-Future.
+/// - red2band_panel_nworkers: number of threads to use for computing the panel in the reduction to band algorithm.
 /// - eigensolver_min_band: The minimun value to start looking for a divisor of the block size.
 ///                         Set with --dlaf:eigensolver-min-band or env variable DLAF_EIGENSOLVER_MIN_BAND.
 /// Note to developers: Users can change these values, therefore consistency has to be ensured by algorithms.
 struct TuneParameters {
+  size_t red2band_panel_nworkers = std::max<size_t>(1, pika::get_num_worker_threads() / 2);
```

Comment:
@rasolca I think we might want to have different defaults for `Backend::MC` and `Backend::GPU` (e.g. half resources on MC and "all" CPU resources on GPU)

Reply:
#803